__mocks__/styleMock.js

```diff
@@ -0,0 +1,73 @@
+/** @flow
```

Comment:
Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

Reply:
This file should actually have been removed: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1215/files

Reply:
Thanks @etoledom, removed!